jobs/envoy/spec

```diff
@@ -16,3 +16,17 @@ consumes:
 - name: frontend_tls_keypairs
   type: frontend_tls_keypairs
   optional: true
+
+properties:
+  statsd.enabled:
+    description: Should a statsd sink be configured
+    default: false
+  statsd.address:
```

Comment:
Does statsd support a DNS name?

Reply:
Oh, I see the default in localhost. So the sink will be colocated, like a BOSH add-on, and be configured independently with the address of the remote aggregator?

Reply:
Exactly. We are using a colocated statsd.